x-pack/plugins/security/server/routes/authentication/saml.ts

```diff
@@ -89,10 +89,10 @@ export function defineSAMLRoutes({
     {
       path: '/api/security/saml/callback',
       validate: {
-        body: schema.object({
-          SAMLResponse: schema.string(),
-          RelayState: schema.maybe(schema.string()),
-        }),
+        body: schema.object(
+          { SAMLResponse: schema.string(), RelayState: schema.maybe(schema.string()) },
+          { unknowns: 'ignore' }
```

Comment:
__note:__ technically it's not related to the main change, but the change is tiny and safe.